pkg/tests/fixtures.go

```diff
@@ -261,6 +264,35 @@ var (
 		},
 	}
 
+	// BookstoreV2TrafficTarget is a traffic target SMI object.
+	BookstoreV2TrafficTarget = target.TrafficTarget{
```

Comment:
Should we add another fixture and test where bookstorev1 and bookstorev2 have the same service account ?